sdk/bindings/c/Makefile

```
@@ -0,0 +1,70 @@
+# Makefile for building Plato C bindings shared library
+
+.PHONY: all clean linux macos windows
```

checkmake (0.2.2)

[warning] Missing required phony target "test"

(minphony)